templates/speaker-template.php

```diff
@@ -0,0 +1,134 @@
+		<main id="main" class="site-main">
+
+			<?php
+			/* Start the Loop */
+			while ( have_posts() ) : the_post();
+				$post_id           = get_the_ID();
+				$first_name        = get_post_meta( $post_id, 'wpcsp_first_name', true );
+				$last_name         = get_post_meta( $post_id, 'wpcsp_last_name', true );
+				$full_name         = $first_name . ' ' . $last_name;
+				$title             = get_post_meta( $post_id, 'wpcsp_title', true );
+				$organization      = get_post_meta( $post_id, 'wpcsp_organization', true );
+				$schedule_page_url = get_option( 'wpcs_field_schedule_page_url' );
+				$speaker_page_url  = get_option( 'wpcsp_field_speakers_page_url' );
+
+				function wpcsp_get_social_links() {
+					$social_icons = [];
+					foreach ( [ 'facebook', 'twitter', 'instagram', 'linkedin', 'youtube', 'website' ] as $social_icon ) {
+						$url = get_post_meta( get_the_ID(), 'wpcsp_' . $social_icon . '_url', true );
+						if ( $url ) {
+
+							$social_label = $social_icon;
+							if ( $social_icon == 'website' ) {
+								$social_icon = 'admin-site-alt3';
+							}
+							if ( $social_icon == 'facebook' ) {
+								$social_icon = 'facebook-alt';
+							}
+
+							ob_start();
+
+							echo '<a class="wpcsp-speaker-social-icon-link" href="' . esc_url( $url ) . '" target="_blank" aria-label="' . $social_label . '"><span class="dashicons dashicons-' . $social_icon . '"></span></a>';
+
+							$social_icons[] = ob_get_clean();
+						}
+					}
+
+					return $social_icons;
+				}
+
+				$args     = [
+					'numberposts' => - 1,
+					'post_type'   => 'wpcs_session',
+					'meta_query'  => [ [ 'key' => 'wpcsp_session_speakers', 'value' => $post->ID, 'compare' => 'LIKE' ] ]
+				];
+				$sessions = get_posts( $args );
+				?>
+
+				<article id="post-<?php the_ID(); ?>" <?php post_class(); ?>>
+
+					<div class="entry-content">
+
+						<div class="wpcsp-speaker-grid">
+
+							<?php if ( has_post_thumbnail() ) {
+								the_post_thumbnail( 'full', [ 'alt' => $full_name ] );
+							} ?>
+
+							<div>
+								<h1 class="entry-title"><?php echo esc_html( $full_name ); ?></h1>
+
+								<div class="wpcsp-speaker-details">
+									<?php if ( $title ) {
+										echo '<p class="wpcsp-speaker-title">' . esc_html( $title ) . '</p>';
+									} ?>
+									<?php if ( $organization ) {
+										echo '<p class="wpcsp-speaker-organization">' . esc_html( $organization ) . '</p>';
+									} ?>
+								</div>
+
+								<?php
+								$social_icons = wpcsp_get_social_links();
+								if ( $social_icons ) { ?>
+									<ul class="wpcsp-speaker-social">
+										<?php foreach ( $social_icons as $social_icon ) { ?>
+											<li class="wpcsp-speaker-social-icon"><?php echo $social_icon; ?></li>
+										<?php } ?>
+									</ul>
+								<?php } ?>
+
+								<h2>About <?php echo esc_html( $full_name ); ?></h2>
+
+								<?php the_content(); ?>
+
+								<?php if ( $sessions ) { ?>
+									<h2>Sessions</h2>
```

Comment:
Translations, wonder if we need a method for the labels like we have for TEC for Events, Venues and Organizers.

Reply:
Yeah, that is a good thing to ticket.